src/xunit.console.netcore/Visitors/StandardOutputVisitor.cs

```diff
@@ -93,9 +96,28 @@ protected override bool Visit(ITestSkipped testSkipped)
 
         protected override bool Visit(ITestStarting testStarting)
         {
+            if (showProgress)
+            {
+                lock (consoleLock)
+                {
+                    Console.Error.WriteLine("   {0} [STARTING]", Escape(testStarting.Test.DisplayName));
+                }
```

Comment:
Yes I figured you were following the precedent which is why I asked if you knew why that is. I personally think stdout would make more since for this given that it isn't actually an error.